scripts/enable_lambdas.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import ast
+from azul import config
+import boto3
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def enable_azul_lambdas(enabled: bool):
+    client = boto3.client('lambda')
+    paginator = client.get_paginator('list_functions')
+    lambda_prefixes = [config.qualified_resource_name(azul_lambda_infix) for azul_lambda_infix in config.lambda_names()]
+
+    assert all(lambda_prefixes)
+
+    for lambda_page in paginator.paginate(FunctionVersion='ALL', MaxItems=500):
+        for lambda_name in [metadata['FunctionName'] for metadata in lambda_page['Functions']]:
+            if any(lambda_name.startswith(prefix) for prefix in lambda_prefixes):
+                lambda_settings = client.get_function(FunctionName=lambda_name)
```

Comment:
```suggestion
                lambda_settings = client.get_function_configuration(FunctionName=lambda_name)
```

is ten times faster.

Reply:
But `client.get_function_configuration` doesn't have the fields for ReservedConcurrentExecutions.

Reply:
After talking with @hannes-ucsc, we are sticking with `client.get_function`, since it doesn't have values for ReservedConcurrentExecutions.